go.mod

```diff
@@ -306,3 +306,5 @@ require (
 )
 
 replace github.com/enterprise-contract/ec-cli/application => ./application
+
+replace github.com/enterprise-contract/enterprise-contract-controller/api => github.com/zregvart/enterprise-contract-controller/api v0.0.0-20231025114412-77586c67f7ca
```

Comment:
Adding a comment here for tracking purposes. We should remove this once https://github.com/enterprise-contract/enterprise-contract-controller/pull/212 is merged. Ideally, before merging this PR.

Reply:
Some might say the commit message is enough, but I'd probably add a comment here too.